examples/image_segmentation/ios/ImageSegmenter/Services/CameraFeedService.swift

```diff
@@ -1,4 +1,4 @@
-// Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+// 2024 The Google AI Edge Authors. All Rights Reserved.
```

Comment:
Copyright is the part that's most important in the header. When I said "The Google AI Edge Authors. All Rights Reserved." I meant that it was just to add The Google AI Edge Authors instead of The TensorFlow Authors since that's the part that could technically change.